client/components/password-change/password-change.js

```diff
@@ -0,0 +1,39 @@
+import "./index.css";
+import React, {useState} from "react";
+
+const PasswordChange = () => {
+  const [oldPassword, changeOldPassword] = useState();
+  const [newPassword, changeNewPassword] = useState();
+
+  const handleSubmit = (oldPassword, newPassword) =>
```

Comment:
Please implement this handleSubmit function. This will make an AJAX call to the node server, which will handle the password change and return some data. If the password is changed successfully  show success message, else show error message. Refer to [this](https://github.com/openwisp/openwisp-wifi-login-pages/blob/299460ed0b73cb9bc03821afed51667561861425/client/components/login/login.js#L50) to get an idea of how it needs to be done.

Reply:
Right. About that, where do I `POST` the form data? And do I run some checks with something like [Joi](https://www.npmjs.com/package/@hapi/joi) before posting? If possible, could you point me to the file which handles the `POST` request on the server/proxy?

Reply:
@someshkar  [here](https://github.com/openwisp/openwisp-wifi-login-pages/blob/299460ed0b73cb9bc03821afed51667561861425/server/routes/account.js#L14) are the routes of the proxy node server. You have to make a post request to the password change route. No, there is no need to use Joi for now.